sdk/monitor/monitor-opentelemetry-exporter/test/internal/trace.exporter.test.ts

```diff
@@ -170,74 +170,6 @@ describe("#AzureMonitorBaseExporter", () => {
           redirectHost
         );
       });
-
-      it("should handle temporary redirects in Azure Monitor", async () => {
```

Comment:
why deleting these tests?

Reply:
Because the core-rest-pipeline is handling 307 and do retry for us, previously we throw an error with status code 307 and remember host in appinsighsClient.host, but since core handles it for us, we get 200 and the test is not fit anymore.